### DISCUSSION TITLE
Timeseries: value mapping colour gets ignored

### DISCUSSION THREAD

**Discussion by natellium** [grafana/grafana#38293] [Dashboards, Panels & Visualizations]
**What happened**:
The colour setting in value mapping doesn't have any effect on the graph
<img width="1427" alt="Screenshot 2021-06-07 at 10 45 36" src="https://user-images.githubusercontent.com/74534993/120987337-c97a3180-c77d-11eb-8060-1d433aab8b1b.png">

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
* Can reproduce here: https://g80-beta.grafana.net/d/NiwIClqGz/simple?orgId=1&editPanel=2
* Create a value mapping with a colour setting in a time series panel

**Environment**:
- Grafana version: 8.0.0-beta3

**Comment by torkelo** [comment]
this is a pretty major feature request ,  unsupported feature. Value mappings color values and the time-series graph does not render values using individual colors, the color is assigned by series (using a palette). 

We are close to supporting color by thresholds in time series, but think color by value mappings will likely never be supported.

**Comment by natellium** [comment]
@torkelo I noticed the colour doesn't have any effect in the case of a table viz either. Is that also a feature request? If so, in which visualisations is the colour option supposed to be currently implemented?

**Comment by torkelo** [comment]
@natellium they do work for table, but you have to also set a cell display mode that uses color (Color text or color background)

**Comment by jrc** [comment]
Thanks for the info. So what are the "Thresholds" colors used for on a Time Series panel then? They don't seem to do anything no matter what I try. 

We're currently using them like this to indicate "healthy" state:
<img width="286" alt="Screen Shot 2021-06-11 at 15 55 12" src="https://user-images.githubusercontent.com/334386/121697544-741b8880-cacd-11eb-9c8b-9421e618a648.png">

**Comment by torkelo** [comment]
In Grafana v8 there is a new option under thresholds, Show as, where you can render thresholds as vertical lines or regions

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
